openmls/src/treesync/node/leaf_node.rs

```diff
@@ -42,6 +39,100 @@ pub(crate) struct NewLeafNodeParams {
     pub(crate) tree_info_tbs: TreeInfoTbs,
 }
 
+#[derive(Debug, PartialEq, Clone)]
+pub(crate) struct SimpleLeafNodeParams {
+    pub(crate) credential_with_key: CredentialWithKey,
+    pub(crate) capabilities: Capabilities,
+    pub(crate) extensions: Extensions,
+}
```

Comment:
What is the point of this type, compared to the regular `LeafNodeParams`? Is it to have something that has all fields set? Can this have a comment?